docs/source/Advanced-Configuration.md

```diff
@@ -47,6 +48,29 @@ docker exec -ti intelowl_uwsgi python manage.py search_index --rebuild
 
 This will build and populate all existing job objects into the `jobs` index.
 
+
+### Business Intelligence
+IntelOwl makes use of [elasticsearch-py](https://elasticsearch-py.readthedocs.io/en/7.x/index.html) to store data that can be used for Business Intelligence purpose.
+Since plugin reports are deleted periodically, this feature allows to save indefinitely small amount of data to keep track of how analyzers perform and user usage.
+At the moment, the following information are sent to elastic:
+- application name
+- timestamp
+- username
+- configuration used
+- process_time
+- status
+- end_time
+- parameters
```

Comment:
could you please provide a sample as an example?

Reply:
![image](https://github.com/intelowlproject/IntelOwl/assets/30528879/a98588f5-414e-40e1-b92a-85f9e2f70aa0)

Reply:
I would use `intelowl-bi` instead of `intel-owl-bi`. In this way we use the `-` correctly.
The environment would be more useful it that was put inside the index name like `intelowl-bi-stag`. In this way we can choose different retention mechanism based on that name (it already works like that for other indexes that we collect)
Is the timestamp related to the moment when this entry is written in elastic or when the analyzer has started?

Reply:
I would save the `application` in lowercase too, just to align with other tags we already save